src/main/java/com/intuit/graphql/orchestrator/schema/transform/FieldResolverTransformerPostMerge.java

```diff
@@ -105,18 +105,13 @@ private DataFetcherContext createDataFetcherContext(final FieldResolverContext f
       String argumentName = inputValueDefinition.getName();
 
       ResolverArgumentDefinition resolverArgumentDefinition = resolverArgumentDefinitionMap.get(argumentName);
-      if (Objects.isNull(resolverArgumentDefinition)) {
-        String errorMessage = String.format("required argument %s not present in resolver argument definition", argumentName);
-        throw new FieldResolverException(errorMessage, fieldResolverContext);
-      }
       ResolverArgumentDefinitionValidator resolverArgumentDefinitionValidator = new ResolverArgumentDefinitionValidator(
               resolverArgumentDefinition, inputValueDefinition, fieldResolverContext
       );
 
       resolverArgumentDefinitionValidator.validate();
 
-      NamedType resolverArgumentType = unwrapAll(inputValueDefinition.getNamedType());
```

Comment:
no need to unwrap here at post merge.  This is needed by `FieldResolverBatchSelectionSetSupplier` to properly convert non primitive NamedType to a Value object.